tools/obscuroscan_v2/frontend/src/stores/priceStore.js

```diff
@@ -9,7 +9,7 @@ export const usePriceStore = defineStore({
         poller: new Poller(() => {
             const store = usePriceStore();
             store.fetch();
-        }, 60*Config.pollingInterval)
+        }, Config.pricePollingInterval)
```

Comment:
Is there a need to have the price updated at the second ? Also, I think it happened to me getting limit rated for polling too fast, might want to avoid the client limit rated for polling too fast too.